BSFAQBOT.py

```diff
@@ -66,7 +70,7 @@ class BOT_DATA:
         'delete': ['delete', 'remove', 'incinerate', 'shred'],
         'edit': ['edit', 'change', 'modify'],
         'recycle': ['recycle', 'bin', 'faq-bin'],
-        'download': ['export', 'download'],
+        'download': ['export'],

```

Comment:
```suggestion
        'download': ['export', 'download'],
```
Any reason for this? I believe this causes `!download` not to work (only `!export`). Why the other commands above all contain the key inside the values, why not this one?